virt/virt-2-4-release-notes.adoc

```diff
@@ -122,6 +122,29 @@ single sidebar menu item labeled *Virtualization*. When you click *Virtualizatio
 //https://bugzilla.redhat.com/show_bug.cig?=id=1959237
 * If you upgrade to {VirtProductName} {VirtVersion}, both older and newer versions of common templates are available for each combination of operating system, workload, and flavor. When you create a virtual machine by using a common template, you must use the newer version of the template. Disregard the older version to avoid issues. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1859237[*BZ#1859237*])
 
+* When creating virtual machines using common-templates or the virtual machine wizard, the default `terminationGracePeriodSeconds` value is set to 0 (force stop).
+This can result in virtual disk failure and can put the virtual workload at risk.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1861297[*BZ#1861297*])
+
+** As a workaround, update the `spec.terminationGracePeriodSeconds` value in the common-templates before creating a virtual machine from that common-template:
++
+. Use the `oc edit` command to edit the common template:
```

Comment:
nit/suggestion: I'd swap this for `Edit the common template by running the following command:`